src/sentry/tasks/deliver_from_outbox.py

```diff
@@ -72,6 +73,14 @@ def schedule_batch(
             scheduled_count += hi - lo + 1
             batch_size = math.ceil((hi - lo + 1) / concurrency)
 
+            metrics.incr(
```

Comment:
```suggestion
            metrics.gauge(
```
Unless I'm misreading this - `metrics.incr` would cause the value to increase by `batch_size` each call

Reply:
Just realized the other parameters would also need to slightly change since gauge has a slightly different signature

Reply:
I wanted it to be an incr since multiple processes can run -- and this is fine, ultimately I'm trying to measure a rate.

Reply:
Or, actually, yeah.... I was originally thinking this should be an incr and a rate, but maybe not.